test/Microsoft.Extensions.Configuration.Test/ConfigurationTest.cs

```diff
@@ -12,6 +12,86 @@ namespace Microsoft.Extensions.Configuration.Test
     public class ConfigurationTest
     {
         [Fact]
+        public void CanChainConfiguration()
+        {
+            // Arrange
+            var dic1 = new Dictionary<string, string>()
+                {
+                    {"Mem1:KeyInMem1", "ValueInMem1"}
+                };
+            var memConfigSrc1 = new MemoryConfigurationProvider(dic1);
+
+            var configurationBuilder = new ConfigurationBuilder();
+
+            // Act
+            configurationBuilder.Add(memConfigSrc1, load: false);
```

Comment:
I didn't actually know this, but `Load()` is a no-op in the base `ConfigurationProvider` and `MemoryConfigurationProvider` doesn't bother overriding it, so the `load: false` used in the test is completely superfluous.

At first glance it doesn't look like anything but unit tests is using this overload of `ConfigurationBuilder.Add()` and BTW `IConfigurationBuilder` doesn't define it. 

 I filed #351 to remove it.